docs/requirements.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright 2024 SECO Mind Srl
+# SPDX-License-Identifier: Apache-2.0
+
+"Sphinx<7"
```

Comment:
```suggestion
Sphinx<=8
```

Reply:
I tested it on my machine, it appears there is no real reason to keep it to <7
So feel free to remove such constraint

Reply:
updated